### ISSUE TITLE
Error: Could not find or load main class Support.gdlauncher_next.datastore.libraries

### ISSUE THREAD

**Issue by PowerSchill** [killpowa/Twickt-Launcher#1075]
**Describe the bug**

When trying to launch an instance with Forge 37.0.69 and Minecraft version 1.17.1 the following error is presented. 

>ps stderr: Error: Could not find or load main class Support.gdlauncher_next.datastore.libraries
Caused by: java.lang.ClassNotFoundException: Support.gdlauncher_next.datastore.libraries

**To Reproduce**

1. Create a Forge instance for 37.0.69
2. Launch the above instance

**Expected behavior**

Minecraft starts loading

**Possible solutions**

**Screenshots**

<img width="509" alt="image" src="https://user-images.githubusercontent.com/11379992/133936751-5bd689a5-9aa2-464a-8449-f6ca653b6aab.png">

**Operating System:**
- OS: MacOS
- Version: 12.0 Monterey Beta
- Java Version: 16
- Minecraft Version: 1.17.1
- Mod Loader: Forge

**Additional context**

**Comment by Eskaan** [comment]
Yeah, we know that. It’s caused by the space in one of your folders in the path.Temporary Fix: in the settings, move your files to somewhere whit no space in the path. i.e. Documents/GDLauncher/

**Comment by blarfoon** [comment]
It’s fixed in latest beta coming out tomorrow, it’ll come out in stable in a week
